content/docs/openshift/install-kubeflow.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Install Kubeflow"
+description = "Instructions for deploying Kubeflow from the command line"
+weight = 4
++++
+
+This guide describes how to use the `kfctl` CLI to deploy Kubeflow 0.7 on an existing OpenShift 4.2 cluster.
+
+## Prerequisites
+
+### OpenShift 4 cluster
+
+* You need to have access to an OpenShift 4 cluster as `cluster-admin` to be able to deploy Kubeflow.
+* You can use Code Ready Containers](https://code-ready.github.io/crc/) (CRC) to run a local cluster, use [try.openshift.com](https://try.openshift.com) to create a new cluster or use an existing cluster.
```

Comment:
The Markdown for the Code Ready Containers link is broken. Please update: Code Ready -> [Code Ready

Reply:
Oh man...sorry..fixed